lib/koans/10_processes.ex

```diff
@@ -2,17 +2,17 @@ defmodule Processes do
   use Koans
 
   koan "Tests run in a process" do
-    assert Process.alive?(self()) == ___
+    assert Process.alive?(self) == ___
```

Comment:
I noticed in a screencast I was watching today that Chris McCord tends to include empty parens on function calls with no arguments. This goes a little against my aesthetic, but I wanted to mention it because perhaps my statement about this "not being idiomatic" isn't entirely true. What are you thoughts?